requirements-test.txt

```diff
@@ -6,4 +6,5 @@ nose
 nosexcover
 pep8>=1.6.0,<1.7
 pylint>=1.4.3,<1.5
+tox
```

Comment:
I didn't have problem with this before. What's the issue?  Also, this is an unrelated change which I prefer to be separate PR.

Reply:
By default `tox` isn't installed in the virtualenv and since it's not installed on my default python install the unit tests are unable to be run locally. I can remove this from the PR if you like.

Reply:
Removed `tox` from the requirements-test.txt